<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="push_button_choose_image">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>45</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name of the chosen image : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select The Image : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_show_image">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>95</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_first_detection">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>First method</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">shadow_detection</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_button_second_detection">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Second method</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">shadow_detection</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_button_first_removal">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>First method</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">shadow_removal</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_button_second_removal">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Second method</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">shadow_removal</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="push_button_detect_shadows">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Detect shadows</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_remove_shadows">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove shadows</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Shadow detection methods :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Shadow removal methods : </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_box_partial_results">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Show partial results</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="shadow_detection"/>
  <buttongroup name="shadow_removal"/>
 </buttongroups>
</ui>
